.gitignore

```diff
@@ -0,0 +1,132 @@
+# ======================================================================
+# Stuff from came from github defaults
+# https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+# ======================================================================
+
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
+*.suo
+*.user
+*.sln.docstates
+
+*_i.c
+*_p.c
+*.ilk
+*.meta
+*.obj
+*.pch
+*.pdb
+*.pgc
+*.pgd
+*.rsp
+*.sbr
+*.tlb
+*.tli
+*.tlh
+*.tmp
+*.tmp_proj
+*.log
+*.vspscc
+*.vssscc
+.builds
+*.pidb
+*.svclog
+*.scc
+
+# Visual Studio output directories
+Debug*/
+Release*/
+Win32/
+x64*/
+Lib/
+build32/
+build64/
+buildWin/
+Codec-Win/codec_core.dir/
+Codec-Win/CodecHarness.dir/
```

Comment:
Looks like there are a bunch of lines in here for another repo - might be nice to clean it up.